tests/test_sequence.py

```diff
@@ -47,9 +48,19 @@ def reg():
     return layout.rectangular_register(4, 7, prefix="q")
 
 
+@pytest.fixture
+def det_map(reg: Register):
+    return reg.define_detuning_map(
+        {"q" + str(i): (1 / 4 if i in [0, 1, 3, 4] else 0) for i in range(28)}
```

Comment:
Why the long range?

Reply:
Ah, so you want to have 0 explicitly on all others qubit IDs. How about cutting it in half and see if still works? Undefined weigths should be assumed to be 0, after all

Reply:
Yes indeed, let's go for this.